rocky/tools/context_processors.py

```diff
@@ -1,6 +1,17 @@
 from django.conf import settings
 
+from account.models import KATUser
+
 
 def languages(request):
     context = {"languages": [code for code, _ in settings.LANGUAGES]}
     return context
+
+
+def organizations_including_blocked(request):
+    context = {}
+    if isinstance(request.user, KATUser):
+        context["organizations_including_blocked"] = [
+            m.organization for m in request.user.members.select_related("organization")
+        ]
+    return context
```

Comment:
The context processor will use the same request and request.user object than the view, so if we put it as cached property on the user object like it was before we only have to do one database query instead of two.

Reply:
I've moved the OrganizationMember accesses to cached properties in fcfff7014bf5e2a0b371abbb7d8678c7f01df8ae, this results in two fewer SQL queries on the crisis room page.